### PR TITLE
HOTFIX: Content Change

### PR DESCRIPTION
**Pull request by balderdash**
Only contains content change for csedweek.org/csteacher, "Participation Kit - for Computer Science Teachers".

### CODE REVIEW

File: pegasus/sites.v3/csedweek.org/public/csteacher.md [hunk 1]
```diff
@@ -3,48 +3,38 @@ title: 'Participation Kit - for Computer Science Teachers'
...
-- [Recruit your entire school](http://hourofcode.org) to participate in Hour of Code. 
-- [Organize a school assembly](https://hourofcode.com/us/how-to/events) to introduce and promote it.
-- Ask students from your existing CS classes to volunteer to help bring Hour of Code to other classrooms.
-- Define your own creative Hour of Code activity - whether it involves showcasing your students' work, involving students to reach out to the rest of the school, or unplugged exercises that are more inclusive.
-- Use our promotional materials: [handout](https://hourofcode.com/us/promote/resources) you can give to the principal. [Celebrity posters](https://hourofcode.com/us/promote/resources#posters) to hang on the walls. [Videos](https://code.org/educate/inspire) to play in a school assembly. See the full [Hour of Code participation kit for schools](https://hourofcode.com/us/how-to/events).
+- Don't limit yourself to student-guided coding games. Check out [teacher-guided lessons plans](http://code.org/educate/teacher-led) for the Hour of Code, and explore [all tutorials](http://code.org/learn) to find more advanced challenges!
```
Comment (breville) [inline pegasus/sites.v3/csedweek.org/public/csteacher.md]: plural lessons?

File: pegasus/sites.v3/csedweek.org/public/csteacher.md [hunk 2]
```diff
@@ -3,48 +3,38 @@ title: 'Participation Kit - for Computer Science Teachers'
...
+- Recruit your entire school to participate in the Hour of Code using [these handouts and sample emails](https://hourofcode.com/us/promote/resources). Or [organize a school assembly](https://hourofcode.com/us/how-to/events) to introduce and promote the Hour of Code.
+- Help computer science students organize an Hour of Code with other classrooms—spread Hour of Code through a whole school!
```
Comment (breville) [inline pegasus/sites.v3/csedweek.org/public/csteacher.md]: &emdash; maybe?  and maybe spaces around that dash anyway?

**Comment by tanyaparker** [comment]
lgtm

**Comment by tanyaparker** [comment]
Oh actually just noticed some links are /us. Mind if I commit directly to this PR?

File: pegasus/sites.v3/csedweek.org/public/csteacher.md [hunk 3]
```diff
@@ -3,48 +3,38 @@ title: 'Participation Kit - for Computer Science Teachers'
...
+- Hang our [celebrity posters](https://hourofcode.com/us/promote/resources#posters) on your classroom's walls.
+- Play a [video](https://code.org/educate/inspire) in a school assembly. 
+- See the full [Hour of Code participation kit for schools](https://hourofcode.com/us/how-to/events).
 - If you organize a big school-wide event, make sure to [submit it as an organizer](http://hourofcode.com), so we can highlight it on our events page and social media channels.
 
-## 2) Host other CS Ed Week events (instead of, or in addition to the Hour of Code)
+### 2) Host other CS Ed Week events (instead of, or in addition to, the Hour of Code)
 
 - Make sure to see NCWIT's [10 ways to recruit girls to CS](https://www.ncwit.org/resources/top-10-ways-recruiting-high-school-women-your-computing-classes/top-10-ways-recruiting) for ideas on activities to grow participation by women
```
Comment (breville) [inline pegasus/sites.v3/csedweek.org/public/csteacher.md]: nit: inconsistent use of periods at ends of sentences in these bullets.

**Comment by balderdash** [comment]
Go for it Tanya

**Comment by tanyaparker** [comment]
(I can make Brendan's fixes too)

File: pegasus/sites.v3/csedweek.org/public/csteacher.md [hunk 4]
```diff
@@ -3,48 +3,38 @@ title: 'Participation Kit - for Computer Science Teachers'
...
-- Go meet with your middle school to show the course offerings and importance of CS education to them (start with [one of these videos](https://code.org/educate/inspire).
-- Reach out to your local four year CS department chair about having your students visit a university.
+- Reach out to your local four year college's CS department chair about having your students visit the campus.
 - Reach out to your local chamber of commerce to have students visit a local employer that depends on computing professionals.
 
+### 3) Recruit broader support for Computer Science
 
-## 3) Recruit broader support for Computer Science
+- Send [one of these sample emails](https://hourofcode.com/us/promote/resources#sample-emails) to your local representatives, city council, or school board and invite them to visit your school for the Hour of Code. It can help build support for computer science in your area beyond one hour.
+- Promote the Hour of Code in your community. [Recruit a local group](https://hourofcode.com/us/promote/resources#sample-emails)—boy/girl scouts club, church, university, veterans group, labor union, or even some friends. You don't have to be in school to learn new skills.
```
Comment (breville) [inline pegasus/sites.v3/csedweek.org/public/csteacher.md]: same dash comments

**Comment by breville** [comment]
just some minor nits

**Comment by tanyaparker** [comment]
Checked all the links for good measure 👍

**Pull request merged**